package.json

```diff
@@ -28,6 +28,7 @@
     "immutable": "^3.7.6",
     "invariant": "^2.2.1",
     "lodash": "^4.13.1",
+    "pretty-fast": "^0.2.0",
```

Comment:
now that we're building the workers it's possible to require the packages directly instead of importScript.